test/e2e/config/environment.go

```diff
@@ -202,12 +202,6 @@ func (e *Environment) getSuiteInstallOpts(suite string, extra ...helm.Option) []
 	return append(opts, extra...)
 }
 
-// GetSelectedSuiteInstallOpts returns the helm install options for the
-// selected suite, appending additional specified ones.
-func (e *Environment) GetSelectedSuiteInstallOpts(extra ...helm.Option) []helm.Option {
```

Comment:
i skimmed through the [blog post](https://go.dev/blog/deadcode), but it wasn't clear to me how the tool can know there are no consumers of exported functions. could they be in use in other repos? 

i had the same question the exported functions from other files in this PR, but realized they live in the `internal` package, which can't be imported from outside this repo.

Reply:
no, sure it doesn't check that, I did a quick scan through GitHub, the only things I spotted importing `github.com/crossplane/crossplane/test/e2e/...` were forks of Crossplane, but I wasn't extremely thorough in my search as I thought it would be safe to assume no one would import Crossplane just for the test functions 😅  but you are right, it's still part of our public API and I might have been a bit too aggressive 😅 the rest of the stuff outside `test/e2e` is in `internal` packages, so it's safe to delete.